packages/runtime/test/src/lib/asa.ts

```diff
@@ -95,7 +95,7 @@ describe("ASA parser", () => {
         total: 0xFFFFFFFFFFFFFFFFn + 5n,
         decimals: 0
       }
-    };
+    } as any;
```

Comment:
please update this to appropriate type

Reply:
This is fine. It's a test. Actually this was added because in previous implementation were using `Object` as type which is not good so i updated it - [https://github.com/scale-it/algo-builder/pull/287/files#diff-b7dea8e4c54f55543c05d4ed625c45ae0f9df28113e9406df19d814ba25c82bbL46](https://github.com/scale-it/algo-builder/pull/287/files#diff-b7dea8e4c54f55543c05d4ed625c45ae0f9df28113e9406df19d814ba25c82bbL46).

That's why had to add type `any` here. In this test we are validating the field the "total". If to use appropriate type we will need to add all other fields in the definition (which aren't related to test). Let me know if you think we should do that.